microcosm_flask/conventions/logging_level.py

```diff
@@ -69,28 +84,103 @@ class LoggerSchema(Schema):
     children = fields.List(fields.Nested("LoggerSchema"), required=True)
 
 
-class LoggingLevel(object):
-    pass
+class UpdateConditionalLoggingLevelSchema(Schema):
+    duration = fields.Float(required=False, missing=300.0)
+    name = fields.String(required=True)
```

Comment:
I almost feel like we'll want this to be a list for conveniently setting a batch of loggers in one go.